### PR TITLE
Add a reference page for XRLayer

### PR DESCRIPTION
**Pull request by Elchi3**
Spec: https://immersive-web.github.io/webxr/#xrlayer-interface

### CODE REVIEW

File: files/en-us/web/api/xrlayer/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Classes extending `XRLayer`
+
+- {{domxref("XRCompositionLayer")}}
+  - {{domxref("XRCubeLayer")}}
+  - {{domxref("XRCylinderLayer")}}
+  - {{domxref("XREquirectLayer")}}
+  - {{domxref("XRProjectionLayer")}}
+  - {{domxref("XRQuadLayer")}}
+- {{domxref("XRWebGLLayer")}}
+
```
Comment (jpmedley) [inline files/en-us/web/api/xrlayer/index.md:31]:
```suggestion
```

File: files/en-us/web/api/xrlayer/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  - VR
+browser-compat: api.XRLayer
+---
+{{APIRef("WebXR Device API")}} {{secureContext_header}}
+
+The **`XRLayer`** interface is the base class for WebXR layer types. It inherits methods from {{domxref("EventTarget")}}.
+
+## Methods
+
+Inherits methods from {{domxref("EventTarget")}}.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/xrlayer/index.md:20]: This is also how it's been done in many other places.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/xrlayer/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - WebXR
+  - XR
+  - AR
+  - VR
+browser-compat: api.XRLayer
+---
+{{APIRef("WebXR Device API")}} {{secureContext_header}}
+
+The **`XRLayer`** interface is the base class for WebXR layer types. It inherits methods from {{domxref("EventTarget")}}.
+
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrlayer/index.md:17]: It is specced and Oculus ships it. https://immersive-web.github.io/layers/#xrcompositionlayertype

File: files/en-us/web/api/xrlayer/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+---
+{{APIRef("WebXR Device API")}} {{secureContext_header}}
+
+The **`XRLayer`** interface of the [WebXR Device API](/en-US/docs/Web/API/WebXR_Device_API) is the base class for WebXR layer types. It inherits methods from {{domxref("EventTarget")}}.
+## Interfaces based on `XRLayer`
+
+Below is a list of interfaces based on the <code>XRLayer</code> interface.
+
+- {{domxref("XRCompositionLayer")}}
+- {{domxref("XRWebGLLayer")}}
```
Comment (jpmedley) [inline files/en-us/web/api/xrlayer/index.md:22]:
```suggestion
- {{domxref("XRWebGLLayer")}}

```

**Review by jpmedley — CHANGES_REQUESTED** [review]
I missed a blank line. Sorry about that.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**